### PR TITLE
Generalized Additive Models

### PR DESCRIPTION
**Pull request by rasmusagren**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-gam/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
```
Comment (bgruening) [inline recipes/r-gam/bld.bat]: This file can be removed

File: recipes/r-gam/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "1.14"
+
+source:
+  fn: gam_1.14.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/gam_1.14.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/gam/gam_1.14.tar.gz
+
```
Comment (bgruening) [inline recipes/r-gam/meta.yaml]: Can we add a checksum here?

File: recipes/r-gam/bld.bat (hunk 1 above)
Comment (rasmusagren) [inline recipes/r-gam/bld.bat]: Why is that? Should R packages never include a bat file? This was generated with conda skeleton.

Comment (bgruening) [inline recipes/r-gam/bld.bat]: bat is used for windows builds, but bioconda is not building for the Win platform.

**Comment by druvus** [comment]
@rasmusagren I was also working to include r-gam and got it working in #2923.

**Comment by rasmusagren** [comment]
@druvus Great, then I remove this one.

**Pull request closed without merging**